templates/zerver/help/keyboard-shortcuts.md

```diff
@@ -1,196 +1,176 @@
+* **Narrow to next unread topic**: `n`
 
-* **Narrow by topic**: `S` — Narrows the view to show all messages with the
-topic of the selected message (outlined in blue).
+* **Narrow to stream**: `s`
 
-* **Narrow to all private messages**: `P` — Narrows the view to show all of the
-user's private messages.
+* **Narrow to topic**: `S`
 
-* **Narrow to next unread topic**: `n` — Narrows the view to the next unread
-topic in the stream sidebar.
+* **Narrow to all private messages**: `P`
 
-* **Cycle between stream narrows**: `A` and `D` — Allows the user to cycle
-through the narrows showing the messages of a stream according to Stream order
-in the left sidebar. `A` allows the user to navigate to the previous stream
-narrow, and `D` allows the user to navigate to the next stream narrow.
+* **Cycle between stream narrows**: `A` (previous) and `D` (next)
 
-* **Return to home view**: `Esc` or `Ctrl` + `[` — Allows the user to return to
-the Home view, showing all messages in the organization.
+* **Narrow to home view**: `Esc` or `Ctrl` + `[`
 
-## Message actions
+## Composing messages
 
-* **Edit your last message**: `⇽` — Opens the last editable message that the
-user sent in the current view (if any) in the compose box.
+* **Reply to message**: `r` or `Enter` (`Return`) — Reply to the selected
+  message (outlined in blue). Same behavior as clicking on the message.
 
-* **Show message sender's profile**: `u` — Shows a popover with the profile of
-the user that sent the selected message.
+* **Reply to message, mentioning author**: `@`
 
-* **Show images in thread**: `v` — Opens any images or videos (if
-any) embedded in a message or previous messages within the thread using the
-[lightbox viewer](/help/view-an-image-at-full-size).
+* **Reply only to author**: `R`
 
-* **Edit selected message**: `i` then `Enter` (`Return`) — Allows the user to
-edit the selected message (outlined in blue) if the user authored the selected
-message. If the selected message was written by another user, this shortcut will
-enable the user to view the source code of the message.
+* **New stream message**: `c` — For starting a new topic in a stream.
 
-* **Star selected message**: `*` — Allows the user to
-[star](/help/star-a-message) the selected message (outlined in blue).
+* **New private message**: `C`
 
-* **React to selected message with <img alt=":thumbs_up:" class="emoji"
-src="/static/generated/emoji/images/emoji/unicode/1f44d.png"
-title=":thumbs_up:"/>**: ` + ` — Allows the user to react to the selected
-message (outlined in blue) with the <img alt=":thumbs_up:" class="emoji"
-src="/static/generated/emoji/images/emoji/unicode/1f44d.png"
-title=":thumbs_up:"/> (`:thumbs_up:`) emoji.
+### In the compose box
 
-* **Collapse/show selected message**: `-` — Allows the user to
-[collapse](/help/collapse-a-message) and uncollapse the contents of the
-selected message (outlined in blue).
+* **Send message**: `Tab` then `Enter`. Also `Enter` or `Ctrl` + `Enter`,
```

Comment:
`Tab` then `Enter` seems to be specific to when pressing enter to send is disabled.